Git checkout  b feature/ella demarest

edemarest opened the pull request.

Commit:
Update config.json

Commit:
8/13/20

Commit:
feature1

Commit:
Update Feature1.java

Commit:
Update Bot.java

eshaank commented:
From what it looks like it there are merge conflicts that have something to do with your config.json. Other than that, it looks good.

Commit:
9/3/20

Commit:
9/10/20

Commit:
9/10/20

Commit:
9/17/20

Commit:
9/24/20

Fixed bug with !fortune and started compiling a doc with all the fortune variations to be implemented next week

Commit:
10/1/20

Commit:
Almost done!!

Commit:
10/15/20

Commit:
10/22/20

Commit:
10/29/20

Commit:
11/5/20

Commit:
11/12/2020

Commit:
11/19/20

Commit:
11/27/20

Commit:
12/3/20

Started on HungerGames instead of War because War was too simple

Commit:
12/10/20

Commit:
12/17/20

Commit:
12/21/2020

FINALLY fixed the bug with organizing and randomizing the teams. Started on the actual rounds which I will try to finish next time

Commit:
12/24/20

Spent forever trying to fix a bug with the methods and also trying to plan what happens each round lol
also i forgot you could make an arraylist of arraylists and wasted a bunch of time when that was the best solution

Commit:
12/31/20

Continued working on Hunger Games program and finally finished the team initialization

Commit:
1/7/2021

Commit:
1/14/21

to change next time:
make 20 max players and let user enter their own names, and then generate enough to add 20 to the # of players, so rounds are long enough.

create 6 team names

Commit:
1/21/2021

Commit:
1/28/21

Commit:
2/4/21

DaveDLeague committed:
Merge branch 'development' into git-checkout--b-feature/EllaDemarest
